Documentation/ExtensionArchitecture/HowTo/UpdateExtensions/UpdateWizards/_SwitchableControllerActionUpgradeWizard.php

```diff
@@ -66,25 +80,15 @@ private function loadFlexForm(string $flexFormString): array
     public function updateNecessary(): bool
     {
         $queryBuilder = $this->connectionPool->getQueryBuilderForTable(self::TABLE);
-        $queryBuilder
+        $queryBuilder->getRestrictions()->removeAll();
```

Comment:
So, is someone un-deletes an element from recycler, it is also migrated. Perfect